server.js

```diff
@@ -652,6 +652,30 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Flattr integration
+camp.route(/^\/flattr\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var thing = match[1]; // eg, `391332`
+  var format = match[2];
+  var apiUrl = 'https://api.flattr.com/rest/v2/things/' + thing;
+  var badgeData = getBadgeData('flattrs', data);
```

Comment:
I try to avoid brands and use words that everybody can relate to. In this case, `donations` may not be the right term. `contributions`, as suggested above, may be better. But I'm not sure what the figure means, so I'm not sure how to make it clearer what it means. You know more, and I'd love advice.